lib/Dialect/FIRRTL/Transforms/SFCCompat.cpp

```diff
@@ -75,7 +75,7 @@ void SFCCompatPass::runOnOperation() {
     // If the `RegResetOp` has an invalidated initialization and we
     // are not running FART, then replace it with a `RegOp`.
     if (!fullAsyncResetExists &&
-        walkDrivers(reg.getResetValue(), true, false, false,
+        walkDrivers(reg.getResetValue(), true, true, false,
```

Comment:
This is my fault originally... this would be better with the `/*name=*/` pseudo-named arguments so it's clear what these are.